### PR TITLE
Allow usage of options on PK in postgreSQL

### LINKED ISSUE #17: Unable to define DEFAULT for PK with PostegreSQL
Hi, the following example doesn't work as I expected with PostgreSQL

```js
create_table("table") {
    t.Column("id", "uuid", {"primary": true, "default_raw": "uuid_generate_v4()"})
}
``` 

I expected the column to be created as descibed but no option are taken into account when primary is set to true

https://github.com/gobuffalo/fizz/blob/master/translators/postgres.go#L24-L37

We could probably do this in the same fashin that the MySQL driver do, by transforming

```SQL
CREATE TABLE table (
    id     UUID  PRIMARY KEY,
);
```

into

```SQL
CREATE TABLE table (
    id     UUID,
    PRIMARY KEY(id)
);
```

And transform the code to something like this (type check is missing here) : 

```go
cols = append(cols, p.buildAddColumn(c))
if c.Primary {
    cols = append(cols, fmt.Sprintf("PRIMARY KEY(%s)", c.Name))
}
```

What's your opinon on this ? I'm OK to build the PR if needed :)

### PR DESCRIPTION
**Pull request by JulienTant**
Fix #17 

The following example doesn't work as I expected with PostgreSQL

```js
create_table("table") {
    t.Column("id", "uuid", {"primary": true, "default_raw": "uuid_generate_v4()"})
}
``` 

I expected the column to be created as described but no options are taken into account when primary is set to true

This PR tries to fix the issue.

**note:** It may also work on sqlite and coackroach but I have absolutely no skill with them and don't wanna break anything... feel free to contribute to my fork, i can give you access to it so that you can push freely

### CODE REVIEW

File: translators/postgres.go [hunk 1]
```diff
@@ -23,19 +23,17 @@ func (p *Postgres) CreateTable(t fizz.Table) (string, error) {
 	for _, c := range t.Columns {
 		if c.Primary {
```
Comment (stanislas-m) [inline translators/postgres.go]: Can you move the primary check after adding the column itself? That's the way it's done with PostgreSQL, so it should work the same way. :)

File: translators/postgres_test.go [hunk 2]
```diff
@@ -11,7 +11,8 @@ var pgt = translators.NewPostgres()
 func (p *PostgreSQLSuite) Test_Postgres_CreateTable() {
 	r := p.Require()
 	ddl := `CREATE TABLE "users" (
-"id" SERIAL PRIMARY KEY,
+PRIMARY KEY("id"),
```
Comment (stanislas-m) [inline translators/postgres_test.go]: As mentioned previously, the MySQL driver have another output. We should have the same order (column def, then PRIMARY constraint) as in the MySQL driver.

File: translators/postgres.go (hunk 1 above)
Comment (JulienTant) [reply translators/postgres.go]: I was trying to avoid having a second if c.Primary bellow but if it's ok for you when i'm doing that!

**Comment by stanislas-m** [comment]
Thanks for your contribution!

**Pull request merged**